src/ssl/ProxyCerts.h

```diff
@@ -11,22 +11,51 @@
 
 #if USE_OPENSSL
 #include "acl/forward.h"
+#include "acl/Gadgets.h"
+#include "ssl/gadgets.h"
 
 class sslproxy_cert_sign
 {
 public:
-    int alg;
-    ACLList *aclList;
-    sslproxy_cert_sign *next;
+    sslproxy_cert_sign() = default;
+    sslproxy_cert_sign(const sslproxy_cert_sign &) = delete; // prohibit all copy/move
```

Comment:
This does [not](https://godbolt.org/z/8Ws4Mnnax) prohibit some copies and some moves. Please delete the move operator instead, as I keep recommending for all such use cases.

```suggestion
    sslproxy_cert_sign(sslproxy_cert_sign &&) = delete; // prohibit all copy/move
```